res/values-cs-rCZ/strings.xml

```diff
@@ -229,6 +229,9 @@
   <string name="confirmation_remove_file_alert">Opravdu chcete odstranit %1$s ?</string>
   <string name="confirmation_remove_folder_alert">Opravdu chcete odstranit %1$s a jeho obsah?</string>
   <string name="confirmation_remove_local">Pouze místní</string>
+  <string name="confirmation_remove_folder_local">Pouze místní</string>
```

Comment:
The message strings shouldn't be modified in pull requests, only the main language (English) can be updated. Our continuous integration server updates on a daily basis all the translations from Transifex. If we update any translation via PRs, the server will fail with a conflict.

Please, remove these added lines, here and in the rest of ```res/values-XX/strings.xml``` files.